electron/main.ts

```diff
@@ -681,6 +681,7 @@ ipcMain.handle('getGameInfo', async (event, appName, runner) => {
       return null
     }
     info.extra = await game.getExtraInfo()
+    // eslint-disable-next-line @typescript-eslint/return-await
```

Comment:
Is this needed?

Reply:
yes, weird, for me it is giving an error that makes no sense.
Let me check again.

Reply:
Yes, removed and the error is gone.
Before it was complaining that I should have an `await` there, but makes no sense and then it was complaining that the `await` was not needed.

Reply:
I have this error sometimes if i switch branches. I think eslint or vscode is sometimes slow.
I started to use `git clean -dxf` on every branch switch.